tests/metrics/utils/test_prompy_template.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+
+from mlflow.metrics.utils.prompt_template import PromptTemplate
+
+
```

Comment:
also, can we add a couple more tests here? i know the logic is simple but it would be helpful to test things like no variables, multiple occurrences of variables, unexpected variables, etc.